moveit_ros/planning/plan_execution/src/plan_execution.cpp

```diff
@@ -266,7 +266,16 @@ bool plan_execution::PlanExecution::isRemainingPathValid(const ExecutableMotionP
 {
   // check the validity of the currently executed path segment only, since there could be
```

Comment:
I don't see this function being used anymore and it is private. Do we still need to keep it as it's just a wrapper?

Reply:
You're right, I just removed it.

I'm so used to MoveIt exposing every single function, it surprised me to see this one (which might actually be useful to an external user) was actually private. :smile: